cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,41 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import ArticlePageObject from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
 
-  });
+const articlePage = new ArticlePageObject();
+const signInPage = new SignInPageObject();
 
-  beforeEach(() => {
-    cy.task('db:clear');
+describe('Article', () => {
+  before(() => {
+    cy.visit('/');
+    signInPage.visit();
+    signInPage.typeEmail('riot@qa.team');
+    signInPage.typePassword('12345Qwert!');
+    signInPage.clickSignInBtn();
   });
 
-  it('should be created using New Article form', () => {
-
+  it.only('should be created using New Article form', () => {
```

Comment:
The use of `it.only` will cause only this test to run, skipping others. If this is not intentional, consider removing `.only` to ensure all tests are executed.